pkg/objstore/inmem/inmem.go

```diff
@@ -105,3 +140,8 @@ func (b *Bucket) Delete(_ context.Context, name string) error {
 	delete(b.objects, name)
 	return nil
 }
+
+// IsObjNotFoundErr returns true if error means that object is not found. Relevant to Get operations.
+func (b *Bucket) IsObjNotFoundErr(err error) bool {
+	return err == errNotFound
```

Comment:
You can create a new type based on error and do a type assertion

Reply:
yea, we can just it is more complex. Will do.

Reply:
actually `inmem` is only used in tests, so I would vote for keeping it simple.